src/meltano/core/migration_service.py

```diff
@@ -62,7 +62,7 @@ def ensure_migration_needed(
             if rev.revision == target_revision:
                 raise MigrationUneededException
 
-    def upgrade(  # noqa: WPS213, WPS231 too many expression and too complex
+    def upgrade(  # too many expression and too complex
```

Comment:
```suggestion
    def upgrade(
```

Reply:
Wdyt we should do with these type of comments?

Reply:
I think they're harmless. It's now unclear that we wrote them in response to linter issues, but that's probably okay.